lib/records/hospital_exporter.rb

```diff
@@ -0,0 +1,131 @@
+module Synthea
+  module Output
+    module HospitalExporter
+      def self.export
+        hospital_list = Synthea::Hospital.hospital_list
+
+        if Synthea::Config.exporter.fhir.export || Synthea::Config.exporter.fhir.upload
+          fhir_record = convert_to_fhir(hospital_list)
+
+          if Synthea::Config.exporter.fhir.upload
+            fhir_upload(fhir_record, Synthea::Config.exporter.fhir.upload)
+          end
+
+          if Synthea::Config.exporter.fhir.export
+            out_dir = get_output_folder('fhir')
+            data = fhir_record.to_json
+            out_file = File.join(out_dir, 'hospital_information.json')
+            File.open(out_file, 'w') { |file| file.write(data) }
+          end
+        end
+
+        if Synthea::Config.exporter.text.export
+          text_record = convert_to_text(hospital_list)
+          out_dir = get_output_folder('text')
+          out_file = File.join(out_dir, 'hospital_information.txt')
+          File.open(out_file, 'w') { |file| file.write(text_record) }
+        end
+      end
+
+      def self.convert_to_fhir(hospital_list)
+        fhir_record = FHIR::Bundle.new
+        fhir_record.type = 'collection'
+        resource_id = SecureRandom.uuid
+
+        hospital_list.each do |h|
+          hospital_resource = FHIR::Organization.new(
+            'id' => resource_id,
```

Comment:
Each Organization in the list shouldn't have the same `resource_id` -- you need to generate a new UUID on each loop.